content/release-10-16-0/whatsnew-10-16-0.md

```diff
@@ -5,3 +5,24 @@ layout: bundle
 ---
 
 Release 10.16.0 includes the following new features or major feature enhancements.
+
+### Multiple LNS Connections for Loriot
+
+The Loriot agent now supports the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+
+![All devices](/images/release-notes/multiple_lns_connection_loriot.png)
+
+For details, refer to [Loriot Multiple LNS Connection](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)
+
+### Support for Loriot Device Registration
+
+Using the Loriot LNS Connections created in the **Connectivity** page in **Settings** under **Administration**, devices can now also be registered using the **Device Registration** page. 
+
+![Device Registration](/images/release-notes/loriot-registration.png)
+
+For Details refer to [Loriot Device Registration](https://cumulocity.com/guides/protocol-integration/lora-loriot/#configure-loriot-credentials-cumulocity)
+
+For the devices registered with the Loriot agent, downlink operations can be sent from the **Shell** tab
+
+For Details refer to [Loriot Downlink Operations](https://cumulocity.com/guides/protocol-integration/lora-loriot/#operations-loriot)
```

Comment:
```suggestion
For details refer to [Sending operations](https://cumulocity.com/guides/protocol-integration/lora-loriot/#operations-loriot).
```

Reply:
@ashhSAG93 Please make sure to use the exact headings in links.

Reply:
Sure @BeateRixen we will check the links and make the changes